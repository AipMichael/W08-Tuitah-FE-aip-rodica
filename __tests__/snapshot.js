import renderer from 'react-test-renderer';
import Home from '../pages';

describe("", () => {
  describe("", () => {
    test("Then it renders homepage unchanged", () => {
      const tree = renderer.create(<Home />).toJSON();
      expect(tree).toMatchSnapshot(); 

    })
  })
})