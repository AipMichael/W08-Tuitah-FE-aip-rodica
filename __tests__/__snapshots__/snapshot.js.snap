// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`  Then it renders homepage unchanged 1`] = `
<div
  className="container"
>
  <main
    className="main"
  >
    <h1
      className="title"
    >
      Welcome to Tuitah
    </h1>
    <div
      className="inline-logo"
    />
  </main>
  <footer
    className="footer"
  >
    Powered by unicorns
  </footer>
</div>
`;
